internal/pkg/bootstrap/bootstrap.go

```diff
@@ -36,13 +37,16 @@ type deviceConfig struct {
 }
 
 type infraConfig struct {
-	LogLevel string `json:"log_level"`
-	HTTPPort string `json:"http_port"`
-	MqttURL  string `json:"mqtt_url"`
-	EdgexURL string `json:"edgex_url"`
-	NatsURL  string `json:"nats_url"`
+	LogLevel     string        `json:"log_level"`
```

Comment:
Any particular reason for these vars to be exported?

Reply:
of course, because of marshalling